lib/zendesk_apps_tools/deploy.rb

```diff
@@ -62,26 +62,23 @@ def upload(path)
       say_error_and_exit e.message
     end
 
-    def find_app_id
+    def find_app_id(product_name = 'v2') # use the v2 endpoint if no product name is provided
       say_status 'Update', 'app ID is missing, searching...'
-      app_name = get_value_from_stdin('Enter the name of the app:')
+      app_name = get_value_from_stdin('Enter the name of the installed app:')
 
-      all_apps_json = cached_connection.get('/api/apps.json').body
+      installations_json = cached_connection.get("/api/#{product_name}/apps/installations.json").body
 
-      app =
-        unless all_apps_json.empty?
-          json_or_die(all_apps_json)['apps'].find { |app| app['name'] == app_name }
-        end
-
-      unless app
-        say_error_and_exit(
-          "App not found. " \
-          "Please verify that your credentials, subdomain, and app name are correct."
-        )
+      if installations_json.empty?
+        say_error_and_exit "Unable to retrieve installations. Please check your internet connection."
+      else
+        installation = json_or_die(installations_json)['installations'].find { |i| i.dig('settings', 'name') == app_name }
```

Comment:
Travis is failing here because ruby 2 2.1 and 2.6.6 doesn't know what dig is yet.